### PR TITLE
Refactored tools to not utilize manifest.yaml

### PR DESCRIPTION
**Pull request by william-price01**
- [x] I have read and agree to the contributing guidelines for [submitting new pull requests](https://github.com/griptape-ai/griptape?tab=readme-ov-file#submitting-pull-requests).

## Describe your changes
Removed manifest.yaml requirements.
## Issue ticket number and link

----
📚 Documentation preview 📚: https://griptape--1226.org.readthedocs.build//1226/

### CODE REVIEW

**Comment by cjkindel** [comment]
Do we have an Issue or other information on why this change is being made and what its impact is?

**Comment by william-price01** [comment]
> Do we have an Issue or other information on why this change is being made and what its impact is?

@collindutter  requested I remove this from the framework.

**Comment by collindutter** [comment]
@cjkindel We've got an internal issue for this. `manifest.yaml` is not used in any way. It's a file from when we had different plans for Tools.

File: docs/griptape-tools/custom-tools/index.md [hunk 1]
```diff
@@ -4,24 +4,11 @@ Building your own tools is easy with Griptape!
 
 To start, create a directory for your tool inside your project. All tool directories should have the following components:
 
-* `manifest.yml` with a YAML manifest.
 * `tool.py` file with a tool Python class.
 * `requirements.txt` file with tool Python dependencies.
```
Comment (collindutter) [inline docs/griptape-tools/custom-tools/index.md:8]: @william-price01 while we're already in here, can you test if this is true? I think this is optional I could be wrong.

Comment (william-price01) [reply docs/griptape-tools/custom-tools/index.md:8]: Test if requirements.txt is required, if your tool must be called tool.py? 
@collindutter

Comment (collindutter) [reply docs/griptape-tools/custom-tools/index.md:8]: Actually, both would be good to test.

**Comment by vasinov** [comment]
Let's hold off on this one as we might actually start using it for hosted tools.

**Comment by collindutter** [comment]
@vasinov and I talked offline. We're going to move forward with this change. Any hosted tool manifest will be a file unique to that cloud feature, not a part of the framework.

**Review by collindutter — APPROVED** [review]

**Pull request merged**